src/Html.php

```diff
@@ -1263,12 +1263,13 @@ public static function renderTagAttributes(array $attributes): string
      * var_dump($options['class']); // outputs: array('persistent' => 'initial');
      * ```
      *
-     * @param array $options the options to be modified.
+     * @param array|null $options the options to be modified.
      * @param string|array $class the CSS class(es) to be added
-     * @see mergeCssClasses()
-     * @see removeCssClass()
+     * 
+     * {@see mergeCssClasses()}
+     * {@see removeCssClass()}
      */
-    public static function addCssClass(array &$options, $class): void
+    public static function addCssClass(?array &$options, $class): void
```

Comment:
BTW: can we remove reference here? New options could be just returned by method, there is no need to pass options as reference here.